roles/openshift_master/tasks/main.yml

```diff
@@ -57,39 +67,43 @@
   args:
     creates: "{{ openshift_master_policy }}"
   notify:
-    - restart master
```

Comment:
Always confused by the indentation. This looks ugly to me but maybe it should be this way. Any justification why beside readability?